src/components/button/Button.js

```diff
@@ -0,0 +1,44 @@
+import './styles.css';
+
+class Button {
```

Comment:
add 'export' before word 'class' because eslint will be very unhappy

Reply:
This is the only one export from this file so it doesn't make sense for it to be named. I read about it when you pointed it at me last time and in this case there should be a default export.
Link me the source that tells it should be another way around :) 
From me:
https://pretagteam.com/question/prefer-default-export-eslint-error

Reply:
you are right :)